admin_manual/configuration/user/user_provisioning_api.rst

```diff
@@ -245,6 +245,100 @@ XML Output
     <data/>
   </ocs>
 
+Enable User
+-----------
+
+Enables a user on the ownCloud server.
+Authentication is done by sending a Basic HTTP Authorization header.
+
+========================================== ======== ==============
+Request Path                               Method   Content Type
+========================================== ======== ==============
+``ocs/v1.php/cloud/users/{userid}/enable`` ``PUT``  ``text/plain``
+========================================== ======== ==============
+
+======== ====== ======================================
+Argument Type   Description
+======== ====== ======================================
+userid   string The id of the user to enable
+======== ====== ======================================
+
+Status Codes
+^^^^^^^^^^^^
+
+* 100 - successful
+* 101 - failure
+
+Example
+^^^^^^^
+
+.. code-block:: console
+
+  # Enable the user ``Frank``
```

Comment:
Double-quotes should be used instead of double-backticks. Sorry, I should have spotted that earlier.